assets/src/components/WarningBanner.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Banner from './Banner';
+
+function WarningBanner (props) {
+    const { children } = props;
+    const defaultMessage = "Something went wrong; please try again later.";
+
+    const bannerSettings = {
+        backgroundColor: "#fff3cd",
+        borderColor: "#ffeeba",
+        textColor: "#856404"
+    };
+
+    return (
+        <Banner settings={bannerSettings}>
+            <>{(children === undefined ? defaultMessage : children)}</>
```

Comment:
I like that fact that you are defaulting to message in case of empty here, can the same approach be adopted to `Alert Banner` component

Reply:
We could, yes, but it's used in a variety of settings so I'm not exactly sure what a good `defaultMessage` would be.